README.md

```diff
@@ -1,3 +1,46 @@
+[![Featurevisor](./assets/banner-bordered.png)](https://featurevisor.com)
+
+<div align="center">
+  <h3><strong>Feature management for developers</strong></h3>
+</div>
+
+<div align="center">
+  <small>Manage your feature flags and experiments declaratively from the comfort of your Git workflow.</small>
+</div>
+
+<br />
+
+<div align="center">
+  <!-- License -->
+  <a href="./LICENSE">
+    <img src="https://img.shields.io/npm/l/@featurevisor/sdk.svg?style=flat-square"
+      alt="License" />
+  </a>
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://featurevisor.com">
+      Website
+    </a>
+    <span> | </span>
+    <a href="https://featurevisor.com/docs">
+      Documentation
+    </a>
+    <span> | </span>
+    <a href="https://github.com/featurevisor/featurevisor-swift/issues">
+      Issues
+    </a>
+    <span> | </span>
+    <a href="https://featurevisor.com/docs/contributing">
+      Contributing
+    </a>
+    <span> | </span>
+  </h3>
+</div>
+
+---
+
 # featurevisor-swift
 
 This repository is a work in progress to port the [Featurevisor](https://featurevisor.com) JavaScript SDK to Swift.
```

Comment:
(not necessarily about this PR, but just to start a conversation)

do you have any opinion about how to handle Swift SDK documentation?

so far, I have tried to put all kinds of docs on the website here: https://featurevisor.com/docs/, which is managed here: https://github.com/fahad19/featurevisor/tree/main/docs

The SDK docs are in one page currently since there's only JavaScript SDK: https://featurevisor.com/docs/sdks/

what could potentially happen is, we start having language specific SDK pages going forward, like:

- `/docs/sdks`: root page listing supported languages
- `/docs/sdks/guide`: how to create a new SDK in a language of your choice
- `/docs/sdks/javascript`: the current `/docs/sdks` page could move here
- `/docs/sdks/swift`: Swift docs can end up here

or, we keep all Swift SDK documentation here in this README in this repository only.

do you have any preference?

Reply:
From what I have seen, the documentation is usually a part of the README for such SDKs, so maybe for now, stick to this. If this will grow or start to being hard to read than we could move it as you pointed out above